app/build.gradle

```diff
@@ -74,7 +74,7 @@ robototextview {
 }
 
 ext {
-    supportLibVersion = '27.1.0'
+    supportLibVersion = '27.1.1'
```

Comment:
The linter was very mad about this because we had conflicting imports from another dependency using 27.1.1 transitively while we were on 27.1.0.